XpringKit/PayID/XRPPayIDClient.swift

```diff
@@ -0,0 +1,67 @@
+import Foundation
+
+/// Provides functionality for XRP in the PayID protocol.
+public class XRPPayIDClient: PayIDClient, XRPPayIDClientProtocol {
+
+  /// The XRP Ledger network that this client attaches to.
+  public let xrplNetwork: XRPLNetwork
+
+  /// Initialize a new XRPPayIDclient
+  ///
+  /// - Parameter xrplNetwork: The XRP Ledger network that this client attaches to.
+  public init(xrplNetwork: XRPLNetwork) {
+    self.xrplNetwork = xrplNetwork
+    super.init(network: "xrpl-\(xrplNetwork.rawValue)")
+  }
+
+  /// Retrieve the XRP address associated with a PayID.
+  ///
+  /// - Note: Addresses are always in the X-Address format.
+  /// - SeeAlso: https://xrpaddress.info/
+  ///
+  /// - Parameter payID: The payID to resolve for an address.
```

Comment:
nit: should the `completion` parameter be mentioned here?

Reply:
double checked with Keefer and he says yes, so I updated that here, in `PayIDclient`'s `address` method and in the `XRPPayIDClientProtocol`.  Thanks Dino!